cli/tauri.js/src/generator.ts

```diff
@@ -0,0 +1,11 @@
+import { writeFileSync } from 'fs-extra'
+import path from 'path'
+import { tauriDir } from './helpers/app-paths'
+import { TauriConfig } from './types/config'
+
+export const generate = (tauriConfig: TauriConfig['tauri']): void => {
+  const { bundle, ...cfg } = tauriConfig
+  const outDir = tauriDir
+  writeFileSync(path.join(outDir, 'config.json'), JSON.stringify(cfg))
```

Comment:
This is where we no longer will need to do this if we use `tauri.conf.json`. @lucasfernog said he would adapt the rust core as soon as you modify the source of the configs from `tauri.conf.js`